asati/cli.py

```diff
@@ -20,7 +24,67 @@ def main(*, verbose: bool) -> None:
     logger.info(configure_sentry())
     logger.info("Running process")
 
-    # Do things here!
+    ssm_client = SSMClient()
+    last_accession_parameter = os.environ["LAST_ACCESSION_PARAMETER"]
+    last_accession_uri_id = ssm_client.get_parameter(last_accession_parameter)
+    logger.info(
+        f"SSM parameter '{last_accession_parameter}' retrieved: '{last_accession_uri_id}'"
+    )
```

Comment:
Given the logging output:

```
INFO asati.utils.get_parameter(): SSM parameter retrieved: 4500
INFO asati.cli.main(): SSM parameter '/apps/asati/last-accession-uri-id' retrieved: '4500'   <-----
```

Maybe this logging line can be removed?  Or reworded to something like "Beginning from accession id: {X}", as we know it's retrieved already, but that logging from the CLI could indicate what we're going to do with it.

Reply:
Good catch, I'll shift the CLI message to `utils` since I think that's the more useful message